### PR TITLE
Task 3963: Work with ButtonGroup as with list

### PR DESCRIPTION
**Pull request by lyubavinaju**
- Delete custom annotation `JDIButtonGroup`
- use `@UI` with buttons locators
- check `ButtonGroup` usages and refactor them if needed
- renamed methods for getting button in ButtonGroup to `button`
- changed and renamed methods in ButtonGroupAssert

### CODE REVIEW

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonGroupAssert.java [hunk 1]
```diff
@@ -21,22 +21,22 @@ public class ButtonGroupAssert extends UISelectAssert<ButtonGroupAssert, ButtonG
      * @param number expected number of buttons
      * @return this {@link ButtonGroupAssert} instance
      */
-    @JDIAction("Number of grouped buttons in '{name}' is '{0}'")
-    public ButtonGroupAssert numberOfGroupedButtons(Integer number) {
+    @JDIAction("Assert that '{name}' has '{0}' buttons")
```
Comment (rvasilyev) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonGroupAssert.java:24]: Может тут кавычки не нужны?

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/ButtonGroup.java [hunk 2]
```diff
@@ -3,41 +3,28 @@
...
      * Gets specific button of this button group using its index.
      *
      * @param index index of button to be found
      * @return button of this button group as {@link Button}
-     * @throws RuntimeException if button with given index not found.
      */
-    @JDIAction("Get Button with index '{0}'")
-    public Button getButtonByIndex(int index) {
-        return castToButton(list().get(index));
+    @JDIAction("Get button with index '{0}'")
```
Comment (rvasilyev) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/ButtonGroup.java:25]: Здесь тоже, раз цифра?

**Review by rvasilyev — CHANGES_REQUESTED** [review]

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonGroupAssert.java (hunk 1 above)
Comment (Kratlis) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonGroupAssert.java:24]: Решили оставить

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/ButtonGroup.java (hunk 2 above)
Comment (Kratlis) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/ButtonGroup.java:25]: Решили оставить

**Review by rvasilyev — APPROVED** [review]

**Review by NewerNaigil — APPROVED** [review]

**Pull request merged**